components/leads-detail.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import Image from 'next/image';
+
+const LeadsModal = () => {
```

Comment:
It is better to keep sync the component name with file name. Here we can rename this component as `LeadDetails`, you can also rename the file name as `lead-details.tsx`.